languages/rust/exercises/concept/structs/.docs/after.md

```diff
@@ -0,0 +1,20 @@
+Notice the usage of `&self` (an _immutable_ reference to our `User` instance) for the getter methods vs. the usage of `&mut self` (a _mutable_ reference to our `User` instance) for the setter methods. The getter methods don't need a mutable reference since their job is to fetch some data; they aren't looking to update or change any fields on the `User`. In contrast, the setter methods _are_ looking to change some data on the `User`, so they require mutable access to the `User` instance. 
```

Comment:
I was slightly confused about the document starting with "Notice the usage of `&self`". I assume you're referring to the exercise? The [`after.md` document](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd) aims to provide a summary of what the student learnt and is a document that aims to suitable for students to check back up on later. As such, the `after.md` document should not be about the exercise, but about the concept taught in the exercise. Does that make sense? For an example, see [this file](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/enums/.docs/after.md).

Note that I think there is some great information in this `after.md` document! It's just that it should not refer to the exercise.

Reply:
The distinction between `&self` and `&mut self` in struct methods is one of the key lessons a student should take away from this exercise, not an incidental.

Reply:
And I'm not trying to say it is :) What I'm saying is that the `after.md` document refers to the exercise, whereas it should not do that as it is a separate document that can also be viewed not only directly after completing an exercise, but far later as a reference. The `after.md` document should definitely contain the information you mentioned, but it shouldn't do that by referring to the exercise.